test/grouping.jl

```diff
@@ -3272,4 +3273,24 @@ end
     @test df == df2
 end
 
+@testset "permutations of operations with combine" begin
+    df = DataFrame(id=rand(1:10, 20))
+    gd = groupby(df, :id)
+    trans = [:id => (y -> sum(y)) => :v1,
+             :id => (y -> 10maximum(y)) => :v2,
```

Comment:
we should add a standard aggregation (as it creates `idx_agg` in a different moment)
```suggestion
             :id => sum => :v3,
```

Reply:
ah - also a no-op like `:id => :v4` might be added